config.md

```diff
@@ -79,8 +79,70 @@ varnish:
 ```
 and the rest will be merged from the defaults.
 
+## Using Wundersecrets
+You can use shared secret variables by providing a private repository into `conf/project.yml`. The repository needs to contain `ansible.yml` in the root folder. Variables from that file are are to wundertools run automatically.
 
+** For example the default Wunder config: **
+```yaml
+wundersecrets:
+  remote: git@github.com:wunderkraut/WunderSecrets.git
+```
+
+## Using upcloud firewall
+Wundertools contains role for setupping firewall rules using [Upcloud Firewall](https://www.upcloud.com/). This reduces load on the machines hosted in Upcloud. We prefer that over `iptables` or `ufw because it can be disabled even if we don't have access to the machines so it's more foolproof and we don't lock us out by accidental changes.
+
+To use upcloud firewall role you need to:
+1. Setup WunderSecrets for the project to access our shared list of allowed IP-addresses
```

Comment:
"Setup" is a noun; "set up" is with a verb (here and elsewhere).

"IP-addresses" -> "IP addresses" (here and elsewhere).